src/super_gradients/training/sg_trainer/sg_trainer.py

```diff
@@ -586,10 +607,49 @@ def train(self, model: nn.Module = None, training_params: dict = None, train_loa
                     where the computed loss is the sum of a few components we would like to log- these entries in
                     loss_items).
 
-                    When training, set the loss_logging_items_names parameter in train_params to be a list of
-                    strings, of length n_items who's ith element is the name of the ith entry in loss_items. Then
-                    each item will be logged, rendered on tensorboard and "watched" (i.e saving model checkpoints
-                    according to it).
+                    IMPORTANT:When dealing with external loss classes, too logg/monitor the loss_items as described
```

Comment:
typo: too -> to

Reply:
fixed in https://github.com/Deci-AI/super-gradients/pull/384/commits/38203b5bcf0daa2f7f26490f85f7eac25636a1c8